### DISCUSSION TITLE
Unify the naming "settlement" vs "collaborative close"

### DISCUSSION THREAD

**Discussion by klochowicz** [get10101/itchysats#742] [General]
There is a mixture of the 2 spread around the codebase, spreading needless confusion. Let's pick one of the 2 and roll it out (preferrably at a time when there's no hure PRs in-flight :) )

Any preference?

**Comment by DeliciousHair** [comment]
Incorporating this into #357

**Comment by thomaseizinger** [comment]
With https://github.com/itchysats/itchysats/discussions/741 in mind, my vote goes down for "settlement" because then we have two fairly distinct names for the two different kinds of closing which I think is good. Features that work completely differently should have different names.
